app.py

```diff
@@ -1,19 +1,59 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import requests as Requests
+import secrets
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    item_list = session.get_items()
+    item_list = []
+    list_id = '5f56323626c33d81cd98b386'
+    r = Requests.get('https://api.trello.com/1/lists/{}/cards?key={}&token={}'.format(list_id, secrets.KEY, secrets.TOKEN))
+    for items in (r.json()):
```

Comment:
You mean `for item in r.json()`
`items` sounds like a list but it is actually a single item because you are looping through the list one item at a time.

Reply:
Done